yaksh/evaluator_tests/test_python_evaluation.py

```diff
@@ -2,52 +2,115 @@
 import os
 from yaksh.python_code_evaluator import PythonCodeEvaluator
 from yaksh.settings import SERVER_TIMEOUT
+from textwrap import dedent
 
 class PythonEvaluationTestCases(unittest.TestCase):
     def setUp(self):
         self.language = "Python"
         self.test = None
-        self.test_case_data = [{"func_name": "add", 
-                                 "expected_answer": "5", 
-                                 "test_id": u'null', 
-                                 "pos_args": ["3", "2"], 
+        self.test_case_data = [{"func_name": "add",
+                                 "expected_answer": "5",
+                                 "test_id": u'null',
+                                 "pos_args": ["3", "2"],
                                  "kw_args": {}
                                 }]
-        self.timeout_msg = ("Code took more than {0} seconds to run. "
-                            "You probably have an infinite loop in your code.").format(SERVER_TIMEOUT)
 
     def test_correct_answer(self):
-        user_answer = "def add(a, b):\n\treturn a + b"""
-        get_class = PythonCodeEvaluator(self.test_case_data, self.test, self.language, user_answer, ref_code_path=None, in_dir=None)
-        result = get_class.evaluate()
+        user_answer = dedent("""
+        def add(a, b):
+            return a + b
+        """)
+        get_evaluator = PythonCodeEvaluator(self.test_case_data, self.test, self.language, user_answer, ref_code_path=None, in_dir=None)
+        result = get_evaluator.evaluate()
```

Comment:
Wrap lines to 80 cols.  Why are you passing the ref_code_path etc. when they are the default values!  Remove them.